conf.py

```diff
@@ -16,8 +16,15 @@
 
 # -- GENERAL -------------------------------------------------------------
 
+def command_output(cmd):
+    output = subprocess.check_output(cmd.split(' '))
```

Comment:
I think it makes more sense to hand-craft the argument lists here, but if you have to use strings, you should use https://docs.python.org/3/library/shlex.html#shlex.split to split them.

Reply:
I guess you refer to shlex.split for command parsing due to problems like this with split: https://github.com/python-telegram-bot/python-telegram-bot/issues/568